Tests/SentryTests/SentryClientTests.swift

```diff
@@ -176,7 +176,9 @@ class SentryClientTest: XCTestCase {
     }
     
     func testCaptureErrorWithoutAttachStacktrace() {
-        let eventId = fixture.getSut().capture(error: error, scope: scope)
+        let eventId = fixture.getSut(configureOptions: { options in
+            options.attachStacktrace = true
```

Comment:
`m`: I think this is a small typo here. I wonder why the tests are passing.
```suggestion
            options.attachStacktrace = false
```

Reply:
Needs to be reviewed then before merging